nitransforms/cli.py

```diff
@@ -0,0 +1,122 @@
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from textwrap import dedent
+import sys
+
+from .linear import load as linload
+from .nonlinear import load as nlinload
+
+
+def cli_apply(pargs):
```

Comment:
Could you add tests for this?. I would probably mock/monkeypatch xfm.apply to just check the inputs are what you expect.